gnoland/website/main.go

```diff
@@ -81,6 +89,73 @@ func handlerFaucet(app gotuna.App) http.Handler {
 	})
 }
 
+func handlerTxs(app gotuna.App) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		var params map[string]json.RawMessage
+		err := json.NewDecoder(r.Body).Decode(&params)
+		if err != nil {
+			writeError(w, fmt.Errorf("%s, %s", "unmarshaling json params", err.Error()))
+			return
+		}
+
+		txData, ok := params["tx"]
+		if !ok {
+			writeError(w, fmt.Errorf("%s, %s", "Missing tx param", err.Error()))
+			return
+		}
+
+		txBz, _ := txData.MarshalJSON()
+		var tx std.Tx
+		err = amino.UnmarshalJSON(txBz, &tx)
+		if err != nil {
+			writeError(w, fmt.Errorf("%s, %s", "unmarshaling tx json bytes", err.Error()))
+			return
+		}
+
+		cli := createClient()
+		res, err := broadcastHandler(cli, tx)
+		if err != nil {
+			writeError(w, err)
+			return
+		}
+
+		if res.CheckTx.IsErr() {
+			writeError(w, fmt.Errorf("transaction failed %#v\nlog %s", res, res.CheckTx.Log))
+			return
+		}
+
+		if res.DeliverTx.IsErr() {
+			writeError(w, fmt.Errorf("transaction failed %#v\nlog %s", res, res.DeliverTx.Log))
+			return
+		}
+
+		result := BroadcastResult{
+			TxHash:    fmt.Sprintf("%X", res.Hash),
+			Height:    res.Height,
+			GasWanted: res.DeliverTx.GasWanted,
+			GasUsed:   res.DeliverTx.GasUsed,
+		}
+
+		w.Header().Set("Content-Type", "application/json")
+		w.WriteHeader(http.StatusOK)
+		json.NewEncoder(w).Encode(result)
+	})
+}
+
+func broadcastHandler(cli client.ABCIClient, tx std.Tx) (*ctypes.ResultBroadcastTxCommit, error) {
+	bz, err := amino.Marshal(tx)
+	if err != nil {
+		return nil, fmt.Errorf("%s, %s", err.Error(), "remarshaling tx binary bytes")
```

Comment:
```suggestion
		return nil, fmt.Errorf("%s, %s", "remarshaling tx binary bytes", err.Error())
```
to keep consistency with the other errors from above?